docs/eventbus/kafka.md

```diff
@@ -44,9 +109,9 @@ named as follows.
 
 ## Horizontal Scaling and Leader Election
 
-Sensors that use a Kafka EventBus can scale horizontally. Unlike NATs and
-Jetstream, specifiying a replicas value greater than 1 will result in all
-Sensor pods actively processing events. However, an EventSource that uses a
-Kafka EventBus cannot be horizontally scaled and a
-[kubernetes leader election](https://argoproj.github.io/argo-events/eventsources/ha/#kubernetes-leader-election) is
-used.
+Sensors that use a Kafka EventBus can scale horizontally. Specifiying replicas
+greater than one will result in all Sensor pods actively processing events.
+However, an EventSource that uses a Kafka EventBus cannot necessarily be
+horizontally scaled in an active-active manner, see [EventSource HA](https://argoproj.github.io/argo-events/eventsources/ha)
```

Comment:
Use relative links, they will be converted automatically to avoid dead links in case the doc structure changes.

Reply:
Done 👍